com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Camera/HDCameraUI.Drawers.cs

```diff
@@ -533,8 +536,8 @@ static void Drawer_FieldDepth(SerializedHDCamera p, Editor owner)
         static void Drawer_FieldClear(SerializedHDCamera p, Editor owner)
         {
             EditorGUILayout.PropertyField(p.clearColorMode, clearModeContent);
-            // if(p.clearColorMode.GetEnumValue<HDAdditionalCameraData.ClearColorMode>() == HDAdditionalCameraData.ClearColorMode.BackgroundColor) or no sky in scene
-            EditorGUILayout.PropertyField(p.backgroundColorHDR, backgroundColorContent);
+            if(p.clearColorMode.GetEnumValue<HDAdditionalCameraData.ClearColorMode>() == HDAdditionalCameraData.ClearColorMode.Color)
```

Comment:
why did we change from background color to Color here?
also should it appear/disappear or be grayed out?

Reply:
- In built-in and URP it disapears. Should we change the behaviour?
- `Color` is the new name, background does not exist anymore, I think we changed somehow and we commented it until they did the replacement. But we forgot to remove the comment.

Reply:
It was replaced long time ago because we don't only clear the background. BackgroundColor make sens in a painter rendering algorithm but we don't really do that so it can be misleading.

Could you check with UX for that. Before it was more : we should hide less possible and prefer disabled fields. Then we start to hide in more options as there was too many of them. I am unsur if it is good or not to hide this. We don't hide fields based on other fields most of the time.

Reply:
It's okay to hide the field when the input is not relevant to the current selection. 

We should only be showing + greying out fields when selection is not available but the information on what is being active / used is still necessary.

Reply:
But can we have Background Color indented under Background Type?

Reply:
@TheoWong-pixel , I've updated the picture with the new editor :)